### ISSUE TITLE
Add nodata query parameter

### ISSUE THREAD

**Issue by petrrr** [EIDA/routing#30]
I would like to propose that a `nodata` query parameter as defined by FDSNWS specification is added to the service. This would help in understand behavior especially if used in a browser window, e.g. for debugging.

**Comment by javiquinte** [comment]
I would definitely like to see this implemented. I will send an email before implementing it (in some weeks) to the ETC members to check that no one is against this, as it will require an extension of the specifications.

**Comment by Jollyfant** [comment]
I agree that it can be useful in browsers but principally I think there is a clear distinction between 204 and 404 response codes. One means success without data, and the other indicates that the resource does not exist (client error). If you need to debug just pop open the console anyway. In standardized APIs I would avoid "complexities" like this.

But I'm not against implementing this ;)

**Comment by petrrr** [comment]
Hi @javiquinte!
After some time I step again into this topic, and I rediscovered this issues.
Are there any news on this? Would be really convient for quick tests.

**Comment by javiquinte** [comment]
Hi @petrrr !
I dedicated a few moments during the Christmas holidays to finish this. Sorry that it took so long!

**Comment by javiquinte** [comment]
Implemented in commit 4cb163c5688088a03dd2f0135c9800ff3c460cb1

**Comment by petrrr** [comment]
Thanks very appreciated!
